kinto/core/storage/postgresql/__init__.py

```diff
@@ -243,8 +243,18 @@ def create(self, collection_id, parent_id, record, id_generator=None,
         VALUES (:object_id, :parent_id,
                 :collection_id, (:data)::JSONB,
                 from_epoch(:last_modified))
-        RETURNING id, as_epoch(last_modified) AS last_modified;
         """
+
+        if ignore_conflict:
+            # We use DO UPDATE so that the RETURNING clause works
+            # but we don't update anything and keep the previous
+            # last_modified value already stored.
+            query += """ON CONFLICT (id, parent_id, collection_id) DO UPDATE
+        SET last_modified = EXCLUDED.last_modified
+        """
+
+        query += "RETURNING id, as_epoch(last_modified) AS last_modified;"
```

Comment:
Instead of concatenating, elsewhere in the code base we use a `safeholder`, like `%(on_conflics)s`. Here providing an empty string by default for example.